README.md

```
@@ -1,497 +1,313 @@
 ```
 
-### Linux
+When installing with yarn 4, you need to disable experimental fetch with the following nodejs config.
 
-```bash
-# Debian/Ubuntu
-sudo apt-get install make
-
-# RHEL/CentOS/Fedora
-sudo yum install make
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)